Dockerfile

```diff
@@ -15,15 +15,15 @@ RUN git clone https://github.com/swagger-api/swagger-ui.git /tmp/swagger-ui \
     && rm -rf /tmp/swagger-ui \
     && cp /go/src/github.com/cyverse-de/permissions/index.html /docs/index.html
 
-RUN go get github.com/constabulary/gb/...
-
-RUN cd /go/src/github.com/cyverse-de/permissions && \
-    gb build && \
-    cp bin/permissions-server /bin/permissions
+RUN go install github.com/cyverse-de/permissions/... \
+    && cp /go/bin/permissions-server /bin/permissions
 
 WORKDIR /
 EXPOSE 60000
 
+ENTRYPOINT ["permissions"]
```

Comment:
This'll break the consul configs, I think. The default entrypoint for the base images is `run-service`, and the `PROGRAM=permissions` above will make it exec into that afterwards. (the default CMD is also `--help`, incidentally)